client/app/components/SideNav/index.jsx

```diff
@@ -0,0 +1,149 @@
+// @flow
+import React from 'react';
+import { BrowserRouter as Router, Route, NavLink } from 'react-router-dom';
+import css from './SideNav.scss';
+
+const routes = [
+  {
+    path: '/dashboard',
+    exact: true,
+  },
+  {
+    path: '/moments',
+    exact: true,
+    routes: [
+      {
+        path: '/categories',
+      },
+      {
+        path: '/moods',
+      },
+    ],
+  },
+  {
+    path: '/strategies',
+    exact: true,
+  },
+  {
+    path: '/medication',
+    exact: true,
+  },
+  {
+    path: '/groups',
+    exact: true,
+  },
+  {
+    path: '/allies',
+    exact: true,
+  },
+];
+const navColor = `${css.bg}`;
+const navTwoColor = `${css.secondarybg}`;
+
+export interface Props {
+  color: string;
+}
+
+export interface State {
+  showMenu: boolean;
+}
+
+export class SideNav extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = { showMenu: false };
+  }
+
+  handleClick() {
+    this.setState({ showMenu: true });
+  }
+
+  handleCloseClick() {
+    this.setState({ showMenu: false });
+  }
+
+  render() {
+    return (
+      <Router>
+        <div className={navColor}>
```

Comment:
it it possible to build these iteratively rather than hardcoding all these `li` elements?

Reply:
This file shouldn't be included in this PR. It also looks like you haven't rebased from master in a while.